🚀Preview Release (#127)
KinsonDigital/CASL: Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.11.md

<h1 align="center" style='color:mediumseagreen;font-weight:bold'>
    CASL Preview Release Notes - v1.0.0-preview.11
</h1>

<h2 align="center" style='font-weight:bold'>Quick Reminder</h2>

<div align="center">

As with all software, there is always a chance for issues and bugs, especially for preview releases, which is why your input is greatly appreciated. 🙏🏼
</div>

---

<h2 style="font-weight:bold" align="center">New Features ✨</h2>

1. [#123](https://github.com/KinsonDigital/CASL/issues/123) - Changed the library to use cross-platform paths when dealing with files and directories.

---

<h2 style="font-weight:bold" align="center">Breaking Changes 🧨</h2>

1. [#115](https://github.com/KinsonDigital/CASL/issues/115) - Changed the `Sound` class to `sealed`.
   > **💡** This prevents anybody from inheriting from the class.

---

<h2 style="font-weight:bold" align="center">Internal Changes ⚙️</h2>
<h5 align="center">(Changes that do not affect users.  Not breaking changes, new features, or bug fixes.)</h5>

1. [#116](https://github.com/KinsonDigital/CASL/issues/116) - Updated the project dotnet and language versions.

>    > **💡** Things comes with lots of performance improvements due to dotnet _**v7.0**_

Reviewer: Grammar
```suggestion
   > **💡** This comes with many performance improvements due to dotnet _**v7.0**_
```

   - Updated dotnet from version _**v5.0**_ to _**v7.0**_
   - Updated C# language version from version _**v9.0**_ to _**v11.0**_
2. [#115](https://github.com/KinsonDigital/CASL/issues/115) - Updated the build system from standard workflows to the [KinsonDigital.CICD](https://kinsondigital.cicd) build system.
---

<h2 style="font-weight:bold" align="center">Nuget/Library Updates 📦</h2>

> 1. [#117](https://github.com/KinsonDigital/CASL/issues/117) - Updated the following NuGet packages.

Reviewer: Grammar
```suggestion
1. [#117](https://github.com/KinsonDigital/CASL/issues/117) - Updated the following NuGet packages:
```

>    - Updated the NuGet package **System.IO.Abstractions** from version _**v13.2.33**_ to _**v19.1.5**_
>    - Updated the NuGet package **Microsoft.CodeAnalysis.NetAnalyzers** from version _**v5.0.3**_ to _**v7.0.0**_

Reviewer: Grammar
```suggestion
   - **System.IO.Abstractions** from version _**v13.2.33**_ to _**v19.1.5**_
   - **Microsoft.CodeAnalysis.NetAnalyzers** from version _**v5.0.3**_ to _**v7.0.0**_
```
